scripts/deployTest.ts

```diff
@@ -0,0 +1,425 @@
+import hre, {ethers, upgrades} from "hardhat";
+import {encodeProgramParameters, encodeRoundParameters} from "./utils";
+import {AddressZero} from "@ethersproject/constants";
+
+const STATUS = {
+    PENDING: 0,
+    ACCEPTED: 1,
+    REJECTED: 2,
+    CANCELED: 3,
+};
+
+const buildNewState = (current: bigint, indexes: number[], statusArray: number[]) => {
+    let newState: bigint = current;
+
+    for (let i = 0; i < indexes.length; i++) {
+        const index = indexes[i];
+        const position = (index % 32) * 2;
```

Comment:
There's a bug with this exact function that we fixed in other scripts. I would recommend using this: https://github.com/gitcoinco/statuses-bitmap